test/patterns/assert_in_code/test_assert_in_code.py

```diff
@@ -1,19 +1,30 @@
 # SPDX-FileCopyrightText: Copyright (c) 2019-2025 Aibolit
 # SPDX-License-Identifier: MIT
 
-import os.path
-from pathlib import Path
+from textwrap import dedent
 from unittest import TestCase
 
-from aibolit.patterns.assert_in_code.assert_in_code import AssertInCode
 from aibolit.ast_framework import AST
-from aibolit.utils.ast_builder import build_ast
+from aibolit.patterns.assert_in_code.assert_in_code import AssertInCode
+
+
+def book_content() -> str:
+    return dedent(
+        '''\
+        // SPDX-FileCopyrightText: Copyright (c) 2019-2025 Aibolit
+        // SPDX-License-Identifier: MIT
+
+        class Book {
+          void foo(String x) {
+            assert x != null;
+          }
+        }
+        '''
+    )
 
 
 class AssertInCodeTestCase(TestCase):
-    cur_file_dir = Path(os.path.realpath(__file__)).parent
 
     def test_assert_in_code(self):
-        file = Path(self.cur_file_dir, 'Book.java')
-        ast = AST.build_from_javalang(build_ast(file))
+        ast = AST.from_string(book_content())
```

Comment:
Since the moment of writing the related issue, we actually opted out from introducing `from_string` method towards this pattern:
```
ast = AST.build_from_javalang(build_ast_from_string(content))
```
It is likely we do not need introducing an alternative constructor `from_string`.